src/xdocs/cmdline.xml.vm

```diff
@@ -60,7 +60,7 @@ java -D&lt;property&gt;=&lt;value&gt;  \
         </li>
         <li>
           <code>-f format</code> - Specifies the output
-          format. Valid values: <code>xml</code>, <code>plain</code> for <a
+          format. Valid values: <code>xml</code>, <code>sarif</code>, <code>plain</code> for <a
           href="apidocs/com/puppycrawl/tools/checkstyle/XMLLogger.html">XMLLogger</a>
           and <a href="apidocs/com/puppycrawl/tools/checkstyle/DefaultLogger
           .html">DefaultLogger</a> respectively.
```

Comment:
please add link to SarifLogger here too, so at leat in javadoc of this class user can find sarif specification and details.

Reply:
Done